src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -703,17 +700,30 @@
   {% endif %}
 {% endblock birthday_widget %}
 
-{% block generatable_text_widget %}
-  <div class="input-group">
-    {{- block('form_widget') -}}
-    <span class="input-group-btn ml-1">
-      <button class="btn btn-outline-secondary js-generator-btn"
-              type="button"
-              data-target-input-id="{{ id }}"
-              data-generated-value-length="{{ generated_value_length }}"
-      >
-        {{ 'Generate'|trans({}, 'Admin.Actions') }}
-      </button>
-   </span>
+{% block file_widget %}
+  <div class="custom-file">
+    {% set attr = attr|merge({
+      class: (attr.class|default('') ~ ' custom-file-input')|trim,
+      'data-multiple-files-text': '%count% files'|trans({}, 'Admin.Global'),
+      'data-locale': get_context_iso_code()
+    }) -%}
+
+    {{ form_widget(form, {'attr': attr}) }}
+
+    <label class="custom-file-label" for="{{ form.vars.id }}">
+      {% set attributes = form.vars.attr %}
+
+      {% if (attributes.placeholder is not defined) %}
+
+        {% if (attributes.multiple is not defined) %}
+          {{ 'Choose file'|trans({}, 'Shop.Theme.Actions') }}
```

Comment:
@LouiseBonnard  - does it look okey here? This input will be applied for all pages.

![Screenshot(6)](https://user-images.githubusercontent.com/17051250/55222899-ad7dbb80-5215-11e9-9172-4d96267f2365.png)

Reply:
It can't be in Shop.Theme.Actions if it regards the back office. Here is my suggestion: _Choose a file_ in _Admin.Actions_.

Reply:
@LouiseBonnard So 'Choose a file' in Admin.Actions and for plural 'Choose files' Admin.Actions too?

Reply:
What about a simpler wording that works for every case, like _Choose file(s)_ in _Admin.Actions_ ? And this way, it can solve the previous question (cf. https://github.com/PrestaShop/PrestaShop/pull/12436#discussion_r270332491) and display _%count% file(s)_ in _Admin.Global_.

Reply:
@LouiseBonnard implemented your suggestion